MultiplayerMod/Multiplayer/Commands/Chores/FindNextChore.cs

```diff
@@ -14,15 +14,15 @@ public class FindNextChore : IMultiplayerCommand {
     private string instanceType;
     private int instanceId;
     private int choreId;
-    private string choreName;
+    private string choreType;
     private int choreCell;
 
-    public FindNextChore(string instanceType, int instanceId, int choreId, string choreName, int choreCell) {
-        this.instanceType = instanceType;
-        this.instanceId = instanceId;
-        this.choreId = choreId;
-        this.choreName = choreName;
-        this.choreCell = choreCell;
+    public FindNextChore(FindNextChoreEventArgs findNextChoreEventArgs) {
```

Comment:
Maybe just `@event` or `args` or whatever shorter?
I mean, it's just an opinion, to use "the last entity" in names of non-common types if there are no similar entities. In case of `FindNextChoreEventArgs` there is an action `FindNextChore`, an event `Event` and arguments `Args`, so it would be `args`. And if there's a similar enity, like `WhateverEventArgs` then, yeah, full name is required to avoid ambiguity.
Makes things more clear.